web/src/components/netflow-record/record-field.tsx

```diff
@@ -469,18 +485,37 @@ export const RecordField: React.FC<{
         }
         return singleContainer(child);
       }
-      case ColumnsId.flowdir:
-        return singleContainer(
-          simpleTextWithTooltip(
-            value === FlowDirection.Ingress
-              ? t('Ingress')
-              : value === FlowDirection.Egress
-              ? t('Egress')
-              : value === FlowDirection.Inner
-              ? t('Inner')
-              : t('n/a')
-          )
-        );
+      case ColumnsId.flowdir: {
+        return singleContainer(simpleTextWithTooltip(getFlowDirectionDisplayString(String(value) as FlowDirection, t)));
+      }
+      case ColumnsId.flowdirints: {
+        if (
+          flow.fields.Interfaces &&
+          flow.fields.FlowDirections &&
+          flow.fields.Interfaces.length === flow.fields.FlowDirections.length
```

Comment:
why u need this check its not possible the two arrays of different len ?

Reply:
It's not supposed to happen indeed, but just in case, I'm checking the length to avoid crashing in the loop following this test since I'm accessing values by their index.